nevergrad/optimization/test_callbacks.py

```diff
@@ -33,9 +33,9 @@ def test_log_parameters(tmp_path: Path) -> None:
     logs = logger.load_flattened()
     assert len(logs) == 32
     assert isinstance(logs[-1]["1"], float)
-    assert len(logs[-1]) == 30
+    assert len(logs[-1]) == 31
```

Comment:
this is normal, you added a parameter to the optimizer, it is recorded in the logs